server/models/Message.js

```diff
@@ -1,17 +1,24 @@
 const mongoose = require("mongoose");
 
 const messageSchema = new mongoose.Schema({
-        sender: {
-          type: mongoose.Schema.Types.ObjectId,
-          required: true,
-          ref: "User",
-        },
-        content: {
-          type: String,
-          default: "",
-        },
-      },
-{conversationTime: true}
-);
+  senderId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "Profile",
+  },
+  receiverId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "Profile",
+  },
+  message: {
+    type: String,
+    default: "",
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now(),
+  },
```

Comment:
You can add the mongoose `timestamps` option to automatically manage the `createdAt` and `updatedAt` fields.